### PR TITLE
Update Description

### LINKED ISSUE #17: change description to reflect new (broader) scope
@cjvanlissa @brandmaier

### PR DESCRIPTION
**Pull request by aaronpeikert**
Just to reflect our new collaboration. This has time, but I'd like your opinion on it @brandmaier.
closes #17

### CODE REVIEW

File: README.Rmd [hunk 1]
```diff
@@ -21,14 +21,14 @@ knitr::opts_chunk$set(
 [![Project Status: WIP – Initial development is in progress, but there has not yet been a stable, usable release suitable for the public.](https://www.repostatus.org/badges/latest/wip.svg)](https://www.repostatus.org/#wip)
 <!-- badges: end -->
 
-The goal of repro is to make the setup of a reproducible workflow with RMarkdown, Git, Make & Docker as easy as possible. To that end it builds upon the great [`usethis`-package](https://github.com/r-lib/usethis). The reproducible workflow mirrors the one proposed by [Peikert & Brandmaier (2019)](https://psyarxiv.com/8xzqy/).
+The goal of `repro` is to make the setup of reproducible workflows as easy as possible. To that end it builds upon the great [`usethis`-package](https://github.com/r-lib/usethis). As `repro` curently undergoes major changes to support more workflows then just the one proposed by [Peikert & Brandmaier (2019)](https://psyarxiv.com/8xzqy/), you may want to take a look at [`worcs`](https://github.com/cjvanlissa/worcs).
```
Comment (cjvanlissa) [inline README.Rmd:24]: I would wait a week, and then say: You may want to install the worcs package on CRAN for a user-friendly workflow that meets most requirements of reproducibility and open science.

This is the benefit of getting worcs on CRAN now: We can start accumulating users before everything is totally perfect.

File: DESCRIPTION [hunk 2]
```diff
@@ -1,6 +1,6 @@
 Package: repro
 Type: Package
-Title: Easy Setup of a Reproducible Workflow
+Title: Automated Setup of a Reproducible Workflow
```
Comment (cjvanlissa) [inline DESCRIPTION:3]: I think the phrase "A reproducible workflow" creates the impression that it's not as customizable as it really is. I would say something like: Automated setup of solutions/components/dependencies/? for reproducible workflow**S**

Comment (aaronpeikert) [reply DESCRIPTION:3]: True, do you like "Automated Setup of a Reproducible Workflows and their components"?
I don't fancy the "of" chain.

File: README.Rmd (hunk 1 above)
Comment (aaronpeikert) [reply README.Rmd:24]: I like it.

File: DESCRIPTION (hunk 2 above)
Comment (cjvanlissa) [reply DESCRIPTION:3]: Automated setup of reproducible workflows and (their dependencies|required tools)?

**Comment by brandmaier** [comment]
Good to go.

**Pull request merged**